NitroxClient/GameLogic/Spawning/Metadata/Processor/EatableMetadataProcessor.cs

```diff
@@ -10,12 +12,32 @@ public override void ProcessMetadata(GameObject gameObject, EatableMetadata meta
     {
         if (gameObject.TryGetComponent(out Eatable eatable))
         {
-            eatable.SetDecomposes(true);
-            eatable.timeDecayStart = metadata.TimeDecayStart;
+            ProcessMetadata(eatable, metadata);
         }
-        if (gameObject.TryGetComponent(out LiveMixin liveMixin))
+        else
         {
-            Resolve<LiveMixinManager>().SyncRemoteHealth(liveMixin, 0);
+            Log.Error($"[{nameof(EatableMetadataProcessor)}] Could not find {nameof(Eatable)} on {gameObject.name}");
         }
     }
+
+    private void ProcessMetadata(Eatable eatable, EatableMetadata metadata)
+    {
+        if (eatable.TryGetComponent(out LiveMixin liveMixin) && liveMixin.health > 0)
```

Comment:
It might happen that we're directly applying the metadata on a living creature. But this metadata is only applied to dead creatures so we first need to kill it the right way so that we can later apply the decay timing correctly